_posts/2025-10-12-newpipe-turns-10.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: "Newpipe is turning 10 :O"
+short: "10 year anniversary"
+date: 2025-10-12 10:00:00 + 01:00
+categories: [pinned, announcement]
+author: <a href="https://github.com/theScrabi">@theScrabi</a> and <a href="https://github.com/Poolitzer">@Poolitzer</a>
+image: birthday-cake
+excerpt_separator: <!-- more -->
+---
+
+Guess what?
+NewPipe just turned 10 ~~this month~~ last month! Time just flew by.
+In one moment you hit “new project”, and in the next there is a whole association, a group of maintainers, an uncountable amount of contributors, and millions of users[^1]!
+This is amazing. Last time we wrote about an important birthday NewPipe just turned five years old. Now its yet another five years later, and we still stand on the ground with both feed, running forward like there is no other direction!
+
+NewPipe is 10 years old \o/, a two digits number. Just... let that settle in for a moment.
+
+<!-- more -->
+## Looking back
+
+5 years ago, Schabi wrote a blog post [for NewPipe's birthday](/blog/pinned/announcement/newpipe-turning-5/). We were right in the beginning of the second wave of the COVID-19 pandemic. Back then he wrote that "People keep working on it and organize the project like COVID-19 doesn’t exist". This is exactly what happened to us in the last five years, and we are so proud of ourselves. Since then many things happened. People from the Project founded and association ([an e.V. in Germany](https://newpipe-ev.de/)) to support NewPipe and other projects like it. We started our [refctoring/rewrite](https://github.com/TeamNewPipe/NewPipe/tree/refactor), with plans for new extractor design and a new player ([NewPlayer](https://github.com/TeamNewPipe/NewPlayer)).
+
+The association on the other hand became a beast on its own. Its goal, to care for the donation money that came in from NewPipe, was far surpassed. In 2024 the association hired [@Profpatsch](https://github.com/Profpatsch) and Schabi to work as part-time workers on several topics regarding the NewPipe refactoring. For this, the association built out a structure and process for hiring, which it achieved greatly. What does that mean for you? Well NewPipe and hopefully other projects gain stability through that, as with paid personal we can assure continuous development even for parts of the project volunteering maintainers will not have a lot of fun with.
```

Comment:
```suggestion
The association on the other hand became a beast on its own. Its goal, to care for the donation money that came in from NewPipe, was far surpassed. In 2024 the association hired [@Profpatsch](https://github.com/Profpatsch) and [Schabi](https://github.com/theScrabi) to work part-time on several topics regarding the NewPipe refactor. For this, the association built out a structure and process for hiring, which turned out successful. What does that mean for you? Well, NewPipe and hopefully other projects gain stability through this, since with paid personnel we can ensure continuous development, even for parts of the project where volunteering maintainers wouldn't have much fun with.
```